freqtrade/optimize/backtesting.py

```diff
@@ -116,7 +116,19 @@ def backtest(args) -> DataFrame:
     records = []
     trades = []
     trade_count_lock: dict = {}
-    exchange._API = Bittrex({'key': '', 'secret': ''})
+
+    # Monkey patch config
+    from freqtrade import main
+    exchange_config = main._CONF['exchange']
+
+    exchange_name = exchange_config['name']
```

Comment:
Not a comment for this PR in particular. But I think the bot should provide decouple methods for loading the config then main/backtesting/hyperopt will use them.